CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 - Bugfix: Fix errors in the API not properly returning a JSON response. (#1833)
 - Bugfix: Fix command "Check message" option not being modifiable. (#1845)
 - Bugfix: Work around no VIPs being refreshed through VIP refresh module in some cases. (#1862)
+- Dev: Add deprecation messages to `add_win` & `remove_win` functions.
```

Comment:
Add PR number here please

Reply:
Will do when stable release is made! good catch

Reply:
Fixed here: https://github.com/pajbot/pajbot/pull/1942/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR32

Reply:
Perfect! Thanks